app/src/main/java/com/github/sdp/ratemyepfl/model/serializer/ItemSerializer.kt

```diff
@@ -0,0 +1,34 @@
+package com.github.sdp.ratemyepfl.model.serializer
+
+import com.github.sdp.ratemyepfl.model.items.Reviewable
+import kotlinx.serialization.SerializationException
+import kotlinx.serialization.json.Json
+
+/**
+ * Defines the serialization strategies for an item. Please use these functions as JSON.encodeToString
+ * may fails for unknown reason.
+ */
+object ItemSerializer {
```

Comment:
Nice, you finally made the polymorphic serialization work !

Reply:
I suffered from this :')